Maple2Storage/Json/GlobalEvents.json

```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "Id": 1,
+    "FirstHour": 0,
+    "FirstMinutesOnHour": 5,
+    "MinutesToRunPerDay": 90,
+    "Events": [
+      12,
+      6,
+      1
```

Comment:
I agree using the enum type would be better, but I would like to not hard code this. This should be flexible for server users to modify in case they want to add or manipulate the standard events.